src/features/dashboard/components/ApiTokenTable/index.tsx

```diff
@@ -54,54 +52,42 @@ const tableColumns: TTokenColumn[] = [
 const ApiTokenTable = (props: HTMLAttributes<HTMLDivElement>) => {
   const { tokens, isLoadingTokens } = useApiToken();
   const { deviceType } = useDeviceType();
+  const is_desktop = deviceType === 'desktop';
 
   const renderTable = () => {
-    return deviceType === 'desktop' ? (
+    return is_desktop ? (
       <Table data={tokens} columns={tableColumns} parentClass='api_token_table' />
     ) : (
       <ResponsiveTable tokens={tokens} />
     );
   };
 
-  const ResponsiveDevice = () => {
-    return clsx(styles.api_table, {
-      [styles.tablet]: deviceType === 'tablet',
-      [styles.mobile]: deviceType === 'mobile',
-    });
-  };
-
   return (
-    <div className={ResponsiveDevice()}>
-      <div style={{ height: `auto` }} className={styles.api_table_container}>
-        <div {...props}>
-          <div className={styles.api_table__header}>
-            <div className={styles.api_table__header__texts}>
-              <Heading.H3>API token manager</Heading.H3>
-              <Text size='md'>Access all your API token details here.</Text>
-            </div>
-            <div className='button-wrap'>
-              <Button
-                color='coral'
-                size='lg'
-                variant='primary'
-                role='submit'
-                iconPosition='start'
-                icon={<LabelPairedCirclePlusMdRegularIcon />}
-                className={styles.api_table__header__button}
-                data-testid='create-new-token-button'
-                onClick={() => {
-                  //
-                }}
-              >
-                Create new token
-              </Button>
-            </div>
-          </div>
-
-          {tokens?.length ? renderTable() : null}
-          {isLoadingTokens && <Spinner />}
+    <div className={styles.api_table}>
+      <div className={styles.api_table__header}>
+        <div className={styles.api_table__header__texts}>
+          <Heading.H3>API token manager</Heading.H3>
+          <Text size='md'>Access all your API token details here.</Text>
         </div>
+        <Button
+          color='coral'
+          size='lg'
+          variant='primary'
+          role='submit'
+          iconPosition='start'
+          icon={<LabelPairedCirclePlusMdRegularIcon />}
+          className={styles.api_table__header__button}
+          data-testid='create-new-token-button'
+          onClick={() => {
+            //
+          }}
```

Comment:
@agrim-deriv I believe you are already handling this click event in your PR, right?

Reply:
Yes @shafin-deriv